config/crd/bases/slo.koordinator.sh_nodemetrics.yaml

```diff
@@ -52,6 +52,14 @@ spec:
                           type: string
                         type: array
                     type: object
+                  nodeMemoryCollectPolicy:
+                    description: NodeMemoryPolicy represents apply which method collect
+                      memory info
+                    enum:
+                    - usageWithHotPageCache
```

Comment:
Please define the available nodeMemoryCollectPolicy in constants, so the controller and the koordlet can use the constant directly. You can follow the definitions of `CalculatePolicy`.